web/src/app/shared/components/nav-bar/nav-bar.component.html

```diff
@@ -1,78 +1,48 @@
-            routerLinkActive="active">Classifications</span>
-        </div>
-        <div class="row submenu" [ngClass]="{'selected': selectedRoute.indexOf('access-items-management') !== -1}">
-          <span (click)="toggleNavBar()" class="col-xs-6" routerLink="taskana/administration/access-items-management"
-            aria-controls="Access items" routerLinkActive="active">Access items</span>
-        </div>
-      </div>
-      <div *ngIf="monitorAccess" class="row menu" [ngClass]="{'selected': selectedRoute.indexOf('monitor') !== -1}">
-        <span (click)="toggleNavBar()" routerLink="{{monitorUrl}}" aria-controls="Monitor" routerLinkActive="active">Monitor</span>
-      </div>
-      <div *ngIf="workplaceAccess" class="row menu" [ngClass]="{'selected': selectedRoute.indexOf('workplace') !== -1 || selectedRoute === ''}">
-        <span (click)="toggleNavBar()" routerLink="{{workplaceUrl}}" aria-controls="Workplace" routerLinkActive="active">Workplace</span>
-      </div>
-      <div *ngIf="historyAccess" class="row menu" [ngClass]="{'selected': selectedRoute.indexOf('history') !== -1}">
-        <span (click)="toggleNavBar()" routerLink="{{historyUrl}}" aria-controls="history" routerLinkActive="active">History</span>
-      </div>
-    </div>
+    </a>
+    <mat-nav-list>
+      <a mat-list-item [routerLink]="'taskana/administration/workbaskets'" [routerLinkActive]="['active']"
+        *ngIf="administrationAccess" (click)="sidenav.toggle()">Administration</a>
+      <a mat-list-item [routerLink]="'taskana/administration/workbaskets'" [routerLinkActive]="['active']"
+        class="list-pos" *ngIf="administrationAccess" (click)="sidenav.toggle()">Workbaskets</a>
+      <a mat-list-item [routerLink]="'taskana/administration/classifications'" [routerLinkActive]="['active']"
+        class="list-pos" *ngIf="administrationAccess" (click)="sidenav.toggle()">Classifications</a>
+      <a mat-list-item [routerLink]="'taskana/administration/access-items-management'" [routerLinkActive]="['active']"
+        class="list-pos" (click)="sidenav.toggle()">Access Items</a>
+      <a mat-list-item [routerLink]="'taskana/monitor'" [routerLinkActive]="['active']" *ngIf="monitorAccess"
+        (click)="sidenav.toggle()">Monitor</a>
+      <a mat-list-item [routerLink]="'taskana/workplace'" [routerLinkActive]="['active']" *ngIf="workplaceAccess"
+        (click)="sidenav.toggle()">Workplace</a>
+      <a mat-list-item [routerLink]="'taskana/history'" [routerLinkActive]="['active']" *ngIf="historyAccess"
+        (click)="sidenav.toggle()">History</a>
+    </mat-nav-list>
     <div class="nav-version">
       <p id="taskana-version"> Taskana version: {{version}} </p>
     </div>
-  </div>
-  <div *ngIf="showNavbar" class="backdrop" (click)="toggleNavBar()"></div>
-</nav>
+  </mat-sidenav>
+
+  <mat-sidenav-content>
+    <nav class="navbar no-border-radius navbar-inverse no-gutter col-xs-12">
```

Comment:
would be cool if you can remove bootstrap classes and replace them with your own css so we don't have to touch navbar again when removing bootstrap completely.

Reply:
Shall I replace the whole navbar then? Since <nav> is a bootstrap component. That's also the reason why I left the css.

Reply:
you can use native CSS instead of bootstrap classes is what I mean. <nav> is a native HTML tag, if you mean class name "navbar" then it is a bootstrap class which is basically something like
{
position: fixed;
height: 55px;
width: 100vw; 
}